docs/Help.md

```diff
@@ -68,18 +68,28 @@ If you're getting the `Login expired for user <user>! Try adding your account ag
 ### How do I add Chatterino as an OBS dock?
 This is not natively supported, but there is a work around:
 
-1. Add a dock into OBS for standard Twitch chat
-2. Turn on `Always on top` in Chatterino
-3. Click the 3 dots up the top of a channel split and click `Popup`
-4. Move and resize the chat to overlay on top of the standard Twitch chat
+1. Add a dock into OBS for standard Twitch chat.
+2. Turn on `Always on top` in Chatterino.
+3. Click the 3 dots up the top of a channel split and click `Popup`.
+4. Move and resize the chat to overlay on top of the standard Twitch chat.
 
-### How do I delete the Chatterino cache?
+### Where is Chatterino folder located?
```

Comment:
There are technically two chatterino folders. Maybe `Where is my Chatterino data folder located?` or something similar.